65black/so-lite-react
# #162832575 Add Delete Question Functionality

#### What does this PR do?
This PR adds the ability for a logged in user to delete their own question by clicking on a button. Other users will not have access to the delete button.

#### Description of Task to be completed?
- add ability for logged in user to delete their question
- add tests for new feature
- switch from axios-mock-adapter to native jest mocking
- update all other tests to reflect change

#### How should this be manually tested?
- Clone the repo, run `npm install` and run `npm start`
- Log in to your account #3 
- Navigate to `localhost:7000/profile` and click on any question
- Scroll down to the end of the question body and click on the red trash icon
- An alert will pop up to confirm your action. Click ok to delete the question
- You should be redirected to the questions page

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#162832575

#### Screenshots (if appropriate)
<img width="1440" alt="screenshot 2018-12-30 at 8 46 59 pm" src="https://user-images.githubusercontent.com/36507972/50550658-cdccc700-0c74-11e9-86e9-be3b8c356ed4.png">

#### Questions:
N/A
